### PR TITLE
Homepage K8s website in Vietnamese

### PR DESCRIPTION
**Pull request by truongnh1992**
Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

File: content/vi/_index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Production-Grade Container Orchestration"
```
Comment (sftim) [inline content/vi/_index.html]: This line is not localized yet

**Comment by truongnh1992** [comment]
> I spotted one line that is not yet localized

Thank @sftim for your review. I've just updated it.

**Comment by sftim** [comment]
#18334 has a fix up to write "Tiếng Việt" rather than "Vietnamese".

**Pull request merged**